doc/menu_lua_api.txt

```diff
@@ -4,99 +4,99 @@ Minetest Lua Mainmenu API Reference 5.7.0
 Introduction
 -------------
 
-The main menu is defined as a formspec by Lua in builtin/mainmenu/
-Description of formspec language to show your menu is in lua_api.txt
+The main menu is defined as a formspec by Lua in `builtin/mainmenu/`
+Description of formspec language to show your menu is in `lua_api.txt`
 
 
 Callbacks
 ---------
 
-core.button_handler(fields): called when a button is pressed.
-^ fields = {name1 = value1, name2 = value2, ...}
-core.event_handler(event)
-^ event: "MenuQuit", "KeyEnter", "ExitButton" or "EditBoxEnter"
+* `core.button_handler(fields)`: called when a button is pressed.
+  * `fields` = {name1 = value1, name2 = value2, ...}
+* `core.event_handler(event)`
+  * `event`: "MenuQuit", "KeyEnter", "ExitButton" or "EditBoxEnter"
 
 
 Gamedata
 --------
 
-The "gamedata" table is read when calling core.start(). It should contain:
-{
+The "gamedata" table is read when calling `core.start()`. It should contain:
+`{
```

Comment:
According to `pandoc -f gfm`, this whole table expression is squashed onto one line. You should use code block syntax. This also goes for some other places where there are multiline table expressions.